pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1489,4 +1489,7 @@ self: super: {
   # Needs a version that's newer than LTS-15.x provides.
   weeder = super.weeder.override { generic-lens = self.generic-lens_2_0_0_0;  };
 
+  polysemy-plugin = super.polysemy-plugin.override {
+    ghc-tcplugins-extra = self.ghc-tcplugins-extra_0_3_2;
+  };
```

Comment:
Could you add a comment here about why this is needed?

Also, could you open an issue upstream about fixing this (if it makes sense)?

If you wanted to get `polysemy-plugin` into Stackage, then you wouldn't have to worry about fixing up this kind of stuff in the future.